src/plugins/embeddable/public/lib/triggers/index.ts

```diff
@@ -17,8 +17,4 @@
  * under the License.
  */
 
-export const CONTEXT_MENU_TRIGGER = 'CONTEXT_MENU_TRIGGER';
-export const APPLY_FILTER_TRIGGER = 'FILTER_TRIGGER';
-export const SELECT_RANGE_TRIGGER = 'SELECT_RANGE_TRIGGER';
-export const VALUE_CLICK_TRIGGER = 'VALUE_CLICK_TRIGGER';
-export const PANEL_BADGE_TRIGGER = 'PANEL_BADGE_TRIGGER';
+export * from './triggers';
```

Comment:
nit: would prefer explicit exports

Reply:
Explicit exports are used at the top level, see: https://github.com/elastic/kibana/pull/57870/files#diff-dd890c48fe59e3e42239eee80458016bR25-R69

Curious, why would you prefer explicit exports here? Namespace exports are easy to maintain and they will show you possible bugs if two types with the same name are exported from different files.

Reply:
We had a sync discussion and agreed to look into a namespace export issues together next time they arise.